### PR TITLE
Specify which collection(s) update/build indexes for

### PR DESCRIPTION
**Pull request by northwestwitch**
Update/build indexes for specific collections, without doing it for all collections.

Example: scout (--demo) index --update -c gene -c transcript --update

**How to test**:
 Locally, we have a goof bug to test the PR with --> #3009
- [ ] From this branch, run the update genes command (`scout --demo update genes`)
- [ ] Note that indexes from genes and transcripts collections are dropped and not recreated
- [ ] Run `scout --demo index  -c gene -c transcript`
- [ ] HAHA, why didn't it work?, repeat the command with the fix
- [ ] The indexes should be created only for those collections

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is very basic, but might work, I'm testing it

**Comment by northwestwitch** [comment]
Ok I feel I'm ready for a review for this one now!

**Comment by northwestwitch** [comment]
Missing indexes in hgnc_genes after update genes:

image

No indexes in tsxs:
image

Run the cmd
image

Indexes are back:
image

image

File: scout/adapter/mongo/index.py [hunk 1]
```diff
@@ -31,15 +31,14 @@ def indexes(self, collection=None):
...
-
-        If this method is utilised when new indexes are defined those should be added
-
+        Args:
+            collections(list): list of collections to update indexes for.
+                               If empty, load indexes for all collections.
         """
-        for collection_name in INDEXES:
+        LOG.info(f"Adding indexes for collections: {collections or list(INDEXES.keys())}")
+        for collection_name in collections or INDEXES:
```
Comment (dnil) [inline scout/adapter/mongo/index.py:41]: 👍

**Review by dnil — APPROVED** [review]

**Pull request merged**